tests/gold_tests/tls/test-0rtt-s_client.py

```diff
@@ -22,25 +22,40 @@
 import os
 import shlex
 import h2_early_decode
+import argparse
 
 
 def main():
-    ats_port = sys.argv[1]
-    http_ver = sys.argv[2]
-    test = sys.argv[3]
-    sess_file_path = os.path.join(sys.argv[4], 'sess.dat')
-    early_data_file_path = os.path.join(sys.argv[4], 'early_{0}_{1}.txt'.format(http_ver, test))
+    parser = argparse.ArgumentParser(description='Process some args.')
+    parser.add_argument('-p', '--ats-port', type=int, dest='ats_port', required=True, help='ATS port number')
```

Comment:
The caller side (tls_0rtt_server.test.py) needs to be updated to use this. https://github.com/apache/trafficserver/pull/9741